portal/server/controllers/utilisation-report-service/utilisation-report-upload/index.js

```diff
@@ -14,7 +14,7 @@ const getUtilisationReportUpload = async (req, res) => {
       const reportPeriod = format(new Date(year, month - 1), 'MMMM yyyy');
       return { ...dueReport, reportPeriod };
     });
-    const { reportPeriod, month, year } = dueReportsWithDetails.at(-1);
+    const { reportPeriod, month, year } = dueReportsWithDetails[0];
     req.session.utilisationReport = { reportPeriod, month, year };
     const nextDueReportDueDate = await getReportDueDate(userToken, new Date(year, month - 1));
```

Comment:
Note to self: this is technically working, but is kind of incorrect

Reply:
This "bug" gets resolved as part of 1089, but I'm going to bring some of that functionality into this ticket